mailbox/client_conn.go

```diff
@@ -122,8 +124,70 @@ func NewClientConn(ctx context.Context, receiveSID,
 		impl:       c,
 		receiveSID: receiveSID,
 		sendSID:    sendSID,
+		serverAddr: serverHost,
+	}
+
+	gbnConn, err := gbn.NewClientConn(
+		gbnN, c.sendToStream, c.recvFromStream,
+		gbn.WithTimeout(gbnTimeout),
+		gbn.WithHandshakeTimeout(gbnHandshakeTimeout),
+		gbn.WithKeepalivePing(gbnClientPingTimeout, gbnPongTimeout),
```

Comment:
which line? Line 134 is 77 chars in my IDE

Reply:
but anyways, i have updated it now so that the gbn options are a variable in ClientConn so that they dont have to be repeated here 👍 so hopefully the line length was sorted along the way 😊